lib/codeowners/config.rb

```diff
@@ -40,10 +40,34 @@ def default_owner=(name)
       @git.config('user.owner', name)
     end
 
+    def default_organization
+      config_org = @git.config('user.organization')
+      return config_org.strip unless config_org.nil? || config_org.strip.empty?
+
+      parse_organization_from_origin
+    end
+
+    def default_organization=(name)
+      @git.config('user.organization', name)
+    end
+
     def to_h
       {
-        default_owner: default_owner
+        default_owner: default_owner,
+        default_organization: default_organization
       }
     end
+
+    protected
+
+    def parse_organization_from_origin
+      origin_url = @git.config('remote.origin.url')
+      return '' if origin_url.nil? || origin_url.strip.empty?
```

Comment:
Maybe we can return nil if we cannot parse 

```suggestion
      return if origin_url.nil? || origin_url.strip.empty?
```

and then only override in the top of the method call:
```ruby
parse_organization_from_origin || ''